deploy/kubectl/openebs-nfs-provisioner.yaml

```diff
@@ -122,6 +122,9 @@ spec:
           value: "true"
         - name: OPENEBS_IO_INSTALLER_TYPE
           value: "openebs-operator-nfs"
+        # OPENEBS_IO_NFS_SERVER_NS defines the namespace for nfs-server deployment
+        #- name: OPENEBS_IO_NFS_SERVER_NS
+        #  value: "default"
```

Comment:
Is the default - `default` namespace or the namespace where openebs is installed.  Probably need to change this "openebs" for better clarity.